core/src/de/jjj/dnasic/PlayerShip.java

```diff
@@ -0,0 +1,45 @@
+package de.jjj.dnasic;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+
+public class PlayerShip extends Sprite {
+
+    private float x;
+    private float y;
+
+    public PlayerShip(TextureRegion shipSkin, float xN, float yN) {
+        super(shipSkin);
+        x = xN;
+        y = yN;
+        super.setPosition(x, y);
+    }
```

Comment:
Maybe rename xN to x and use `this.x = x` to make the parameters more understandable.